src/Transformers/LinkTransfomer.js

```diff
@@ -13,6 +13,6 @@ export const transform = link => ({
   count: link.count,
   url: link.url,
   url_short: new URL(link.key, config('app.url')),
-  updatedAt: link.updatedAt.toISOString(),
-  createdAt: link.createdAt.toISOString(),
+  updated_at: link.updatedAt.toISOString(),
+  created_at: link.createdAt.toISOString(),
```

Comment:
These are not used anywhere yet (since they didn't exist in Bertly 1.0). I'm just renaming for consistency.